pages/common/git-stash.md

```diff
@@ -4,24 +4,24 @@
 
 - Stash current changes (except new files):
 
-`git stash save {{optional_stash_name}}`
+`git stash save {{optional_stash_message}}`
 
 - Include new files in the stash (leaves the index completely clean):
 
-`git stash save -u {{optional_stash_name}}`
+`git stash save -u {{optional_stash_message}}`
 
 - List all stashes:
```

Comment:
Actually stash list does more than listing the names. The man page says:

> Each stash is listed with its name (e.g.  stash@{0} is the latest stash, stash@{1} is the one before, etc.), the name of the branch that was current when the stash was made, and a short description of the commit the stash was based on.
>
>    stash@{0}: WIP on submit: 6ebd0e2... Update git-stash documentation
>    stash@{1}: On master: 9cc0589... Add git-stash

Maybe something like "List all stashes with name, related branch and message"?

And we can even improve the first two examples:

```
- Stash current changes, except new files (default message is the last commit message):

`git stash [save {{optional_stash_message}}]`

- Stash current changes, including new/untracked files (leaves the index completely clean):

`git stash -u`
```

Thoughts?